platform/podjs/src/manifest-parser/src/tests/manifest.test.ts

```diff
@@ -3,15 +3,15 @@ import { join } from "path";
 
 describe("Parsing", () => {
     describe("Successful parse", () => {
-        it.each(["manifest.json", "no-version.json"])("%s", async (pkg) => {
+        it.each(["manifest.json"])("%s", async (pkg) => {
```

Comment:
Again, a problem with this path. These now simply become tests for pod.js, so have you considered moving them to `tests` or `src/tests`?